docs/docs/examples/guides/abb-hardware-interface.md

```diff
@@ -0,0 +1,285 @@
+module corresponds to the UDPUC device configured during step 1 above. The module can be placed in the controller's home
+ directory and uploaded to the current task.
+
+<details>
+  <summary>Example RAPID module</summary>
+
+```bash
+MODULE AICAMain
+  !***********************************************************
+  ! Program data
+  !***********************************************************
+  ! Home position.
+  LOCAL VAR jointtarget current;
+
+  ! Identifier for the EGM correction.
+  LOCAL VAR egmident egm_id;
+
+  ! Limits for convergance.
+  LOCAL VAR egm_minmax egm_condition := [-0.1, 0.1];
+
+  !***********************************************************
+  !
+  ! Procedure main
+  !
+  !   This RAPID code exemplifies how to run EGM with velocity control
+  !
+  !   Note: Update the UCDevice "AICA_PC" with correct
+  !         values for the remote address and port
+  !         (i.e. to the EGM server).
+  !
+  !         Update via RobotStudio:
+  !         Controller tab -> Configuration ->
+  !         Communication -> UDP Unicast Device
+  !
+  !***********************************************************
+  PROC main()
+      current := CJointT();
+      MoveAbsJ current, v200, fine, tool0;
+
+      ! Register an EGM id.
+      EGMGetId egm_id;
+
+      ! Setup the EGM communication.
+      EGMSetupUC ROB_1, egm_id, "default", "AICA_PC", \Joint;
+
+      ! Prepare for an EGM communication session.
+      EGMActJoint egm_id
+                  \J1:=egm_condition
+                  \J2:=egm_condition
+                  \J3:=egm_condition
+                  \J4:=egm_condition
+                  \J5:=egm_condition
+                  \J6:=egm_condition
+                  \MaxSpeedDeviation:=120.0;
+
+      WHILE TRUE DO
+          ! Start the EGM communication session.
+          EGMRunJoint egm_id, EGM_STOP_HOLD, \J1 \J2 \J3 \J4 \J5 \J6 \CondTime:=5 \RampOutTime:=5 \PosCorrGain:=0;
+          EGMStop egm_id, EGM_STOP_HOLD \RampOutTime:=5;
+      ENDWHILE
+      ! Release the EGM id.
+      EGMReset egm_id;
+
+
+  ERROR
+      IF ERRNO = ERR_UDPUC_COMM THEN
+          TPWrite "Communication timed out";
+          TRYNEXT;
+      ENDIF
+  ENDPROC
+ENDMODULE
+```
+
+</details>
+
+## Hardware interface
+
+:::tip
+
+For best results, always set the rate of the hardware interface to 250 Hertz, which corresponds to the ABB-suggested
+stable UDP data exchange limit.
+
+:::
+
+Returning to AICA Studio and the hardware interface, it is now possible to define the parameters and connect to the
+robot. The majority of the hardware interface parameters enable connection to EGM and RWS:
+
+<div class="text--center">
+  <img src={abbHIParameters} alt="ABB Hardware interface parameters." />
+</div>
+
+- EGM port: the port that EGM uses to send commands (6511, the Remote port of the UDPUC device defined in RobotStudio).
+- RWS IP & port: address and port of the RWS (the address of the RobotStudio device or real robot, 192.168.137.1 and
```

Comment:
I mentioned this above, I don't think users always know how to access the logs below easily, so might be better not to show them and instead work with the default values for ports? What do you think?